src/js/model.js

```
@@ -1,8 +1,14 @@
+import {
+    Scene, PerspectiveCamera, WebGLRenderer,
+    AmbientLight, DirectionalLight, Vector3
+} from "three";
```

ESLint

[error] Missing trailing comma.

(comma-dangle)